hastexo/hastexo.py

```diff
@@ -232,10 +240,32 @@ def parse_xml(cls, node, runtime, keys, id_generator):
 
         return block
 
-    def author_view(self, context=None):
```

Comment:
Not quite sure why this method was dropped? Does it no longer apply and if it doesn't, what can the author expect to render in Studio, here?

Reply:
`xblock-utils` uses this view to render the children.  I think overriding it just for the somewhat obvious warning that the terminal doesn't work in Studio is not worth the added maintenance overhead of having to keep the overriden function in sync with `xblock-utils`.